huaweicloud/services/acceptance/workspace/data_source_huaweicloud_workspace_tags_test.go

```diff
@@ -0,0 +1,175 @@
+package workspace
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/acceptance"
+)
+
+func TestAccWorkspaceTagsDataSource_basic(t *testing.T) {
+	var (
+		rName = acceptance.RandomAccResourceNameWithDash()
+
+		desktop      interface{}
+		resourceName = "huaweicloud_workspace_desktop.test"
+		rc           = acceptance.InitResourceCheck(
+			resourceName,
+			&desktop,
+			getDesktopFunc,
+		)
```

Comment:
```suggestion
		resourceName = "huaweicloud_workspace_desktop.test"
		rc           = acceptance.InitResourceCheck(
			resourceName,
			&desktop,
			getDesktopFunc,
		)
```
Delete this part. The data source does not need to check whether the corresponding resource exists.